### PR TITLE
conf(INVT-CPC-874): invt bring up to date project dependencies

### PR DESCRIPTION
**Pull request by nic5694**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CPC-874
## Context:
Updated config files to bring the service up to date on spring boot dependencies.
## Changes
- Updated versions in the config file
## Before and After UI (Required for UI-impacting PRs)
N/A

### CODE REVIEW

File: inventory-service/Dockerfile
Comment (swafit) [inline inventory-service/Dockerfile:1]: The gradle version also has to be changed in the distributionUrl gradle version in `microservice/gradle/gradle-wrapper.properties`

Comment (nic5694) [reply inventory-service/Dockerfile:1]: yeah your right i added it in the following commit

**Review by Eli293 — APPROVED** [review]
Looks good, nicely done.

**Review by Eli293 — APPROVED** [review]
Nice job

**Review by DylanBrass — APPROVED** [review]

**Pull request merged**